apis/placement/v1beta1/binding_types.go

```diff
@@ -42,6 +46,20 @@ type ResourceBindingSpec struct {
 	TargetCluster string `json:"targetCluster"`
 }
 
+// BindingState is the state of the binding
+type BindingState string
+
+const (
+	// BindingStateScheduled means the binding is scheduled but need to be bound to the target cluster.
+	BindingStateScheduled BindingState = "Scheduled"
+
+	// BindingStateBound means the binding is bound to the target cluster.
+	BindingStateBound BindingState = "Bound"
+
+	// BindingStateUnScheduled means the binding is not scheduled on to the target cluster anymore.
+	BindingStateUnScheduled BindingState = "UnScheduled"
```

Comment:
hi ryan, could you clarify that, scheduled = creating, bound = active, unscheduled = deleting?

is my understanding correct?

Reply:
yes